app/controllers/CalculationController.scala

```diff
@@ -39,10 +39,13 @@ class CalculationController @Inject()(cc: MessagesControllerComponents,
                                      ) extends FrontendController(cc) with Logging {
 
   implicit val calcSuccessResponseWrites: OWrites[CalcSuccessReponse] = Json.writes[CalcSuccessReponse]
-  private val calc23and24NinoPrefix : Seq[String] = configuration.getOptional[Seq[String]]("calc23and24NinoPrefix").getOrElse( Seq[String]() )
+
+  // Defines NINOs to be intercepted and response retrieved from ATs stub data
+  val incomeSourcesNinos: Set[String] = Set("AS000001A", "AS000002A", "AS000003A", "AS000004A", "AS000005A",
+    "AS000006A", "AS000007A", "CE453003A", "CE453004A", "MN000001A", "MN000002A")
 
   def generateCalculationListFor2023_24(nino: String): Action[AnyContent] = {
-    if (calc23and24NinoPrefix.exists(prefix => nino.startsWith(prefix))) {
+    if (incomeSourcesNinos.contains(nino)) {
```

Comment:
would actually be nice to allow a list of regex's if poss :)

Reply:
Regex ninos or prefixes? I tried to be explicit here to avoid creating users in future (years down the line maybe) and having something break